oak_runtime/src/introspect.rs

```diff
@@ -95,6 +95,42 @@ fn find_ids(path: &str, kind: &str) -> Option<(u64, u64)> {
     Some((id, subid))
 }
 
+// Looks for a matching file generated by webpack and returns it
+fn find_client_file(path: &str) -> Option<(File, String)> {
+    let filepath = Regex::new(r"^/dynamic/(?P<filepath>[^\s]+)$")
+        .unwrap()
+        .captures(path)?
+        .name("filepath")
+        .unwrap()
+        .as_str();
+
+    // Do not serve paths that go up the tree
+    if filepath.contains("../") {
+        return None;
+    }
+
+    let path_string = format!(
+        "./oak_runtime/src/introspection_browser_client/dist/{}",
```

Comment:
I don't think this can work in general &ndash; a live `oak_loader` binary might be bundled up into (say) a Docker image and executed with no sign of the `oak_runtime/src/` tree being available.

Maybe add a new option to `oak_loader` that gets plumbed through into `RuntimeConfiguration` that indicates a directory (at run-time) which should be served up by the introspection server for paths under `/dynamic/` ?

Reply:
Alternatively bundle the assets in the binary itself, via https://doc.rust-lang.org/std/macro.include_bytes.html or similar. This seems safer than allowing looking up files from the underlying fs anyways.

Reply:
I like bundling the assets, I'll look into that :))

Reply:
Resolved with https://github.com/project-oak/oak/pull/1382/commits/2903b75180fae7d72ba9816b4154220f1a55780a

Reply:
(Unresolving, since there's disagreements around plumbing a directory vs bundling files)

Reply:
> (Personally, I think `include_bytes` is going in the wrong direction – I think the introspection client code should more loosely bound to the Runtime code, not more tightly bound.)
@daviddrysdale regarding your comment from the other thread, if decoupling the two further is the goal, perhaps the the introspection client shouldn't be served the introspection aux server at all? 

Passing a path through the `RuntimeConfiguration` to the aux server seems like a lot of overhead. An alternative could be to independently up a different server all-together in the runner. Eg running something like https://github.com/lukejacksonn/servor on a separate port, which merely reads from the events provided by the introspection aux server. 

Though I'd wager that since the existing introspection logic is already bundled, I'm not sure it's necessary to split the dynamic version of it.

Reply:
Resolving since the discussion moved to another thread